REST_API_DOCS.md

```
@@ -0,0 +1,1205 @@
+
+클라우드 커뮤니티 서비스의 REST API 명세서입니다.
+
+## 사용자(User) API
+
+### { 내가 좋아요한 게시글 목록 }
+
+**GET** `/users/me/likes`
+
+로그인한 사용자가 좋아요를 누른 게시글 목록을 조회한다.
+
+**Request Headers**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)